modules/gapi/include/opencv2/gapi/cpu/gcpukernel.hpp

```diff
@@ -43,13 +43,9 @@ namespace cpu
      * stack. Every backend is hardware-oriented and thus can run its
      * kernels efficiently on the target platform.
      *
-     * Backends are usually "back boxes" for G-API users -- on the API
+     * Backends are usually "black boxes" for G-API users -- on the API
      * side, all backends are represented as different objects of the
-     * same class cv::gapi::GBackend. User can manipulate with backends
-     * mainly by specifying which kernels to use or where to look up
-     * for kernels first.
-     *
-     * @sa @ref gapi_hld, cv::gapi::lookup_order()
```

Comment:
I think it is ok to keep "User can manipulate with backends by specifying which kernels to use." and keep link to hld.

Only lookup_order should go.

Reply:
Fixed